apps/zui/scripts/merge-mac-release-files.mjs

```diff
@@ -0,0 +1,227 @@
+// Based on https://github.com/electron-userland/electron-builder/issues/5592#issuecomment-2004803764
+import { Octokit } from "octokit";
+import pkg from '../package.json' assert {type: 'json'};
+import { TextDecoder } from 'node:util';
+import fs from 'node:fs';
+import { Readable } from 'node:stream';
+import { Buffer } from 'node:buffer';
+import yaml from 'js-yaml';
+
+const token = process.env.GH_TOKEN
+
+const client = new Octokit({
+  auth: token
+});
+
+// These are derived from settings in package.json so the script will work on
+// both regular Zui and Zui Insiders.
+const OWNER = pkg.repository.split('/')[3];
+const REPO = pkg.repository.split('/')[4];
+const PRODUCT_NAME= pkg.productName.replaceAll(' ', '-');
+const REPO = 'zui';
+const URL = `/repos/${OWNER}/${REPO}/releases`;
+const VERSION = pkg.version;
+const RELEASE_NAME = (PRODUCT_NAME == 'Zui') ? 'v' + VERSION : VERSION;
```

Comment:
Here's another coping mechanism. For all the time they've existed, Zui Insiders builds have names _without_ the leading `v` (e.g., [1.7.1-19](https://github.com/brimdata/zui-insiders/releases/tag/v1.7.1-19)) while regular Zui have it (e.g., [v1.7.0](https://github.com/brimdata/zui/releases/tag/v1.7.0)). Here I rationalized leaving things as they are because GitHub already does some annoying things with how it sorts releases, so I didn't want to make things even worse by also changing the naming convention.